portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -26,7 +34,62 @@
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    Query query = new Query("Comment").addSort("timestamp", SortDirection.DESCENDING);
```

Comment:
nit: Since this is the same every time, consider making it a static constant instead of re-creating it on every GET request. In Java this means adding a new `private static final` property to the class